relay-server/src/services/processor/replay.rs

```diff
@@ -146,7 +146,7 @@ fn handle_replay_event_item(
                     if *segment_id > 720 {
                         metric!(counter(RelayCounters::ReplayExceededSegmentLimit) += 1);
 
-                        relay_log::warn!(
+                        relay_log::debug!(
```

Comment:
@cmanallen do you still need this warning for debugging purposes? It's currently producing a lot of log lines.

Reply:
If its causing issues remove it.